src/mvs/mvsNodeActions.ts

```diff
@@ -27,22 +29,30 @@ export async function uploadDialog(node: ZoweDatasetNode, datasetProvider: IZowe
 
     const value = await vscode.window.showOpenDialog(fileOpenOptions);
 
-    await Promise.all(
-        value.map(async (item) => {
-            // Convert to vscode.TextDocument
-            const doc = await vscode.workspace.openTextDocument(item);
-            await uploadFile(node, doc);
-        }
-    ));
+    if (value && value.length) {
```

Comment:
Not sure if `value.length` is required here. I think we cannot pass an empty string via the Open File dialog, but I am not opposed to leave it in as an extra security measure.

Other than this, I have tested the implementation, and works as expected.

Reply:
Should be fine to go after all checks pass